remove stylelint for project and use global config, update readme docs (#36)
sigstore/sigstore-website: README.md

# sigstore-website

![sigstore logo][logo]

[logo]: /assets/icons/logo.svg 'sigstore logo'

## Security

Should you discover any security issues, please refer to sigstore's [security
process](https://github.com/sigstore/community/blob/main/SECURITY.md)

> # sigstore

Reviewer: ```suggestion
# Tooling
```

The sigstore website is run on an open source framework called Nuxt JS, a VUE app framework. You can read more about it [here](https://nuxtjs.org/). The reason we chose this had many factors that included greater SEO support, SPA's, and the ability to connect an API of our choice being easy.

> 

Reviewer: ```suggestion

## Managing the content
```

sigstore content is created with Netlify CMS, which is an open source content management system that generates markdown content. The integration within the Nuxt app was perfect and the query interface for getting content was very clear and easy to set up.

> To log into the CMS you need to be invited via Netlify identity to access the CMS itself. Once you have gone through the necessary steps to creating and verifying your account you can go in and start editing content.

Reviewer: ```suggestion
To log into the CMS you need to be invited via Netlify Identity. Once you've create and verified your account you can go in and start editing content.

### Editing content
The [guide to editing sigstore.dev content is here](/#)
```
